src/strategies.rs

```diff
@@ -122,33 +112,29 @@ mod test {
     use super::ExpBackoffStrategy;
     use std::time::Duration;
 
-    macro_rules! assert_close_to {
-        ($item:expr, $value:expr) => {
-            assert_close_to!($item, $value, 0.001_f64)
-        };
-        ($item:expr, $value:expr, $eps: expr) => {
-            assert!($item >= ($value - $eps));
-            assert!($item <= ($value + $eps));
-        };
-    }
-
     #[test]
     fn test_exponential_backoff_jitter_bounds() {
-        let mut backoff_iter = ExpBackoffStrategy::new(Duration::from_secs(1), 2, 0.1).into_iter();
-        let first = backoff_iter.next();
-        assert_close_to!(first.unwrap().as_secs_f64(), 2.0_f64, 0.2_f64);
-        let second = backoff_iter.next();
-        assert_close_to!(second.unwrap().as_secs_f64(), 4.0_f64, 0.4_f64);
+        let mut backoff_iter = ExpBackoffStrategy::new(Duration::from_secs(1), 2., 0.1)
+            .with_seed(0)
+            .into_iter();
+        for idx in 1..10 {
```

Comment:
because you know the exact seed, isn't this 100% deterministic? Why is an ub and lb needed? Can't you just assert the first 10 items are the exact same 10 values that you are expecting?

Reply:
I think we wanted different tests here. I was just out to test that the resulting values are within the bounds imposed on jitter (hence the name of the test), and not really caring about the specific values. I'll change the test.